tests/x11regressions/libreoffice/libreoffice_mainmenu_favorites.pm

```diff
@@ -14,6 +14,7 @@
 use base "x11regressiontest";
 use strict;
 use testapi;
+use utils;
```

Comment:
See remark about `utils` vs `version_utils` above.

Reply:
Thanks, I just fixed it, please help to review.